src/StartScreen.java

```diff
@@ -12,17 +12,28 @@ public Car showStartCars() {
             System.out.println(i + ". " + startCars.get(i).getModel() + " - " + startCars.get(i).getCarCost() + "$");
             i++;
         }
-        return carChoise();
+        return carChoice(player);
     }
 
-    private Car carChoise() {
+    private Player carChoice(Player player) {
         int i = InOutService.inputInt();
         if (0 < i && i < 4) {
-            System.out.println("Your new car is " + startCars.get(i).getModel());
-            return startCars.get(i);
+            return purchaseStartCar(player,startCars.get(i));
         } else {
-            System.out.println("Wrong choise, try again");
-            return carChoise();
+            System.out.println("Wrong choice, try again");
+            return carChoice(player);
+        }
+    }
+
+    public Player purchaseStartCar(Player player, Car car) {
+        if (player.getMoney() >= car.getCarCost()){
+            player.setMoney(player.getMoney() - car.getCarCost());
+            player.setMyCar(car);
+            System.out.println("Your new car is "+car.getModel());
+            return player;
+        }else{
+            System.out.println("Not enough money, make another choice");
+            return carChoice(player);
         }
```

Comment:
We can use this not only for start car. So we need separate sevice for money operation